charts/super-agent-deployment/templates/configmap-superagent-config.yaml

```diff
@@ -0,0 +1,15 @@
+{{- $name := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "config" ) -}}
+{{- $namespace := .Release.Namespace -}}
+{{- with ( include "newrelic-super-agent.config" . ) -}}
```

Comment:
using `if` instead of `with` could we save two variables?

Reply:
I already did it on [#7ec3d4b](https://github.com/newrelic/helm-charts/pull/1155/commits/7ec3d4ba9073b07f1ca579fc3547ebcb10695eef).